reports/historical/_gmmp_2015_importer.py

```diff
@@ -33,6 +33,21 @@ def slurp_table(
             row_heading_col,
         )
 
+    def import_by_yearly_data(self, yearly_data):
```

Comment:
data should mean _data_... I think you mean _info_ here so may be `import_grid(self, grid_info)` or something